content/spin/v2/rdbms-storage.md

```diff
@@ -113,7 +113,26 @@ export const handleRequest: HandleRequest = async function (request: HttpRequest
 
 {{ startTab "Python"}}
 
-The Python SDK doesn't currently surface the MySQL or PostgreSQL APIs.
+> [**Want to go straight to the reference documentation?**  Find it here.](https://fermyon.github.io/spin-python-sdk/)
+
+The code below is an [Outbound MySQL example](https://github.com/fermyon/spin-python-sdk/tree/main/examples/spin-mysql). There is also an outbound [PostgreSQL example](https://github.com/fermyon/spin-python-sdk/tree/main/examples/spin-postgres) available.
+
+```python
+from spin_sdk import http
+from spin_sdk.http import  Request, Response
```

Comment:
```suggestion
from spin_sdk.http import Request, Response
```

maybe do a quick sweep for the double-space since it looks like it's been copied to several places